package.json

```diff
@@ -103,7 +105,24 @@
     "packages/*"
   ],
   "lint-staged": {
-    "packages/**/demo/**/*.js": ["eslint --fix --format=node_modules/eslint-formatter-pretty", "git add"],
-    "packages/**/modules/**/*.js": ["eslint --fix --format=node_modules/eslint-formatter-pretty", "git add"]
+    "demo/**/*.js": [
```

Comment:
Split's this up into more specific sections to format and lint only where needed. This means that we can: auto-format JS and TS while also formatting documentation and e.g. type-check using flow before commiting.